jta-crash-rec-eap/eap64-qs-test-s2i.json

```diff
@@ -32,21 +32,41 @@
             "displayName": "Git Repository URL",
             "description": "Git source URI for application",
             "name": "SOURCE_REPOSITORY_URL",
-            "value": "https://github.com/jboss-developer/jboss-eap-quickstarts",
+            "value": "https://github.com/ochaloup/openshift-quickstarts",
```

Comment:
I might be misunderstanding, but this should be jboss-openshift/openshift-quickstarts shouldn't it?

Reply:
yes, it should be. Sorry, it was change because of my testing and I forgot to change it back. :innocent:
Thanks for spotting it!